pkg/channel/distributed/receiver/channel/channel.go

```diff
@@ -88,8 +88,8 @@ func ValidateKafkaChannel(channelReference eventingChannel.ChannelReference) err
 		}
 	}
 
-	// Check KafkaChannel READY Status
-	if !kafkaChannel.Status.IsReady() {
+	// Check if KafkaChannel failed
+	if kafkaChannel.IsFailed() {
```

Comment:
I changed it back

Reply:
I still see this as being changed?  the `.Status` part was removed which means it will call the new top-level `IsReady()` which has the generation check.  Am I missing something?